# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/waylandpp/hotfix-cstdint.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

gcc-15 no longer transitively includes <cstdint>, upstream fixed after 1.0.0
Signed-off-by: Dennie Mahne <dennie.mahne@gmail.com>

--- waylandpp-1.0.0/scanner/scanner.cpp
+++ waylandpp-1.0.0/scanner/scanner.cpp
@@ -23,6 +23,7 @@
 #include <vector>
 #include <cctype>
 #include <cmath>
+#include <cstdint>
 #include <stdexcept>

 #include "pugixml.hpp"
